backend/solo_rog_api/views.py

```diff
@@ -46,3 +53,35 @@ class DocumentList(generics.ListAPIView):
         ("statuses__status_date", "last updated"),
     ]
     ordering = ["statuses__status_date"]
+
+
+class D6TSubmissionView(CreateAPIView):
+    serializer_class = UpdateStatusD6TSerializer
+
+    def get_serializer(
+        self, *args: Any, **kwargs: Any
+    ) -> Union[BaseSerializer, BaseSerializer]:
+        return super().get_serializer(many=True, allow_empty=False, **kwargs)
+
+    def get_serializer_context(self) -> Dict[str, Any]:
+        ctx = super().get_serializer_context()
+        ctx["new_status"] = "D6T"
+        ctx["document_excludes"] = {"statuses__dic__code": "D6T"}
+        ctx["document_filters"] = {"statuses__dic__code": "AS2"}
+        return ctx
+
+
+class CORSubmissionView(CreateAPIView):
+    serializer_class = UpdateStatusCORSerializer
+
+    def get_serializer(
+        self, *args: Any, **kwargs: Any
+    ) -> Union[BaseSerializer, BaseSerializer]:
+        return super().get_serializer(many=True, allow_empty=False, **kwargs)
+
+    def get_serializer_context(self) -> Dict[str, Any]:
+        ctx = super().get_serializer_context()
+        ctx["new_status"] = "COR"
+        ctx["document_excludes"] = {"statuses__dic__code": "COR"}
+        ctx["document_filters"] = {"statuses__dic__code": "D6T"}
+        return ctx
```

Comment:
This shouldn't be necessary now that the COR/D6T submit serializers are separate